ginga/misc/plugins/ChangeHistory.py

```diff
@@ -256,3 +256,10 @@ def start(self):
 
     def __str__(self):
         return 'changehistory'
+
+
+# Replace module docstring with config doc for auto insert by Sphinx.
+# In the future, if we need the real docstring, we can append instead of
+# overwrite.
+from ginga.util.toolbox import generate_cfg_example
+__doc__ = generate_cfg_example('plugin_ChangeHistory', package='ginga')
```

Comment:
At its current state. This is what will happen:

Incantation on `.rst` | Incantation on `.py` | Result
---- | ----- | -----
Yes | Yes | Config file shown as expected
Yes | No | Shows actual module docstring instead (not desirable)
No | Yes | No docstring shows
No | No | No docstring shows